src/routing_table/authority.rs

```diff
@@ -46,22 +47,47 @@ pub enum Authority {
         peer_id: PeerId,
         /// The name of the single ManagedNode which the Client connects to and proxies all messages
         /// through.
-        proxy_node_name: XorName,
+        proxy_node_name: N,
     },
 }
 
-impl Authority {
+impl<N: Xorable> Authority<N> {
     /// Returns true if group authority, otherwise false.
     pub fn is_group(&self) -> bool {
         match *self {
             Authority::ClientManager(_) |
             Authority::NaeManager(_) |
             Authority::NodeManager(_) => true,
+            Authority::Section(_) |
             Authority::ManagedNode(_) |
             Authority::Client { .. } => false,
         }
     }
 
+    /// Returns true if section authority, otherwise false.
+    pub fn is_section(&self) -> bool {
+        match *self {
+            Authority::ClientManager(_) |
+            Authority::NaeManager(_) |
+            Authority::NodeManager(_) |
+            Authority::ManagedNode(_) |
+            Authority::Client { .. } => false,
+            Authority::Section(_) => true,
+        }
+    }
+
+    /// Returns `true` if the destination is an individual node, and `false` otherwise.
```

Comment:
I'm fine with either, with a slight preference for putting the backticks (although, TBH, I haven't been paying much attention to it so far).